src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -2205,6 +2233,15 @@ void PolicyHandler::OnAppsSearchCompleted(const bool trigger_ptu) {
   policy_manager_->OnAppsSearchCompleted(trigger_ptu);
 }
 
+void PolicyHandler::OnAddedNewApplicationToAppList(
+    const uint32_t new_app_id, const std::string policy_id) {
```

Comment:
@ydementieiev nice to have auto trace here

Reply:
@AKalinich-Luxoft done in [fe7b393](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/fe7b393fe80f89ac9137d6dd9b41a0dfd6ba28c6)